### DISCUSSION TITLE
Minimal sdk api version for android runtime

### DISCUSSION THREAD

**Discussion by dhirajnitk** [microsoft/onnxruntime#6438] [Other Q&A] [answered]
Nnapi is supported from sdk api #27  in android
But I wanted to know if it's possible to build onnxruntime for older SDK api??

**Comment by guoyu-wang** [comment, marked as answer]
Yes, you can build onnxruntime with NNAPI for API level as low as 24, but if the onnxruntime is used on any device with API level less than 27, NNAPI EP will not be part of the execution.

You can build onnxruntime for Android without NNAPI for API level as low as 21.

**Reply by dhirajnitk** [reply]
How about building onnxruntime on android without nnapi.
Does it support from api 21(+l
